common/changes/@microsoft/rush/keco-npm-access_2019-01-10-23-45.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Add support for NPM --access flag for publish action",
+      "packageName": "@microsoft/rush",
+      "type": "none"
```

Comment:
@iclanton @nickpape-msft @octogonz what is the correct type for this level of change? Type of `minor`?

Reply:
Yeah - `minor`.